### ISSUE TITLE
1.9.5.1 review

### ISSUE THREAD

**Issue by Prulon** [Pz1c/WavingHands#160]
1. [Critical] App icon is wrong. 
Should be the icon with the blue background. Now it is white

2. [High] Text in the review screen is cut. 

3. [Critical] was in previous discussion but not implemented:
I think that for new players, clicking on Start Training should immediately
create a game with a bot. Only when you played a few, and the main button
becomes NEW GAME then we should have the special new screen with the
different options.
Basically I want to make sure that we dont get a drop in players who start
their first game. 
We also said we'll make the NEW GAME show by default when you've finished playing 3 games OR elo > 1500

4. [Critical] invite friend still doesn't work for me. I created TesterP124 --> clicked invite --> saved the link --> signed out --> signin as Prulon --> click on link --> opens play store --> click play --> I don't see any game created

**Comment by Pz1c** [comment]
1. doh, I forget about new icon
2. it's possible to scroll?
3. will be in 1.9.5.2
4. should be fixed under #158

**Comment by Prulon** [comment]
About 2 - try to surrender, you will see that you only get a part of the
sentence like "raising a white" and the nothing

**Comment by Pz1c** [comment]
@Prulon I am sorry but I lost link on new app icon
may I ask you to resend it
thanks

**Comment by Prulon** [comment]
here's the [link](https://drive.google.com/file/d/1GRIfH7QVUsIkuHckcZNHNKwjKBs7Edfr/view?usp=sharing
)

@Pz1c

**Comment by Prulon** [comment]
About #158 I understand you put it under 1.9.6.
So do I understand correctly that the whole "Invite a friend" is currently not working, or does it happen only in some cases?
@Pz1c

**Comment by Pz1c** [comment]
it should work for case when app not installed yet I hope

**Comment by Prulon** [comment]
Ok

**Comment by Pz1c** [comment]
1. done - thanks for link
2. looks like fixed on desktop 
3. done

![image](https://user-images.githubusercontent.com/5511791/151160934-6211c20d-abae-40ff-95f0-fc2b1866b339.png)

**Comment by Pz1c** [comment]
2 looks like fixed on PC
![image](https://user-images.githubusercontent.com/5511791/151162696-24580a8c-9424-4cb6-81ad-c6d49529ab91.png)
(desc for surrendering was changed only for testing ;))

**Comment by Prulon** [comment]
Looks good :)
